LETSENCRYPT.md

```diff
@@ -64,15 +63,12 @@ The contact e-mail for these certificates is configured by the env var
 Certifcates are stored in an AWS Elastic File System mounted into the EC2
 instance at `/etc/letsencrypt`.
 
-Nginx has been configured to reroute requests for `/.well-known` on port 80 
+Nginx has been configured to reroute requests for `/.well-known` on port 80
 to certbot's internal web service. This allows certbot to prove to LetsEncrypt
 that Isomer has control of the domain that we are requesting a certificate for,
 via an [HTTP-01 challenge](https://letsencrypt.org/docs/challenge-types/).
 
-Custom SSL certificates are still supported by adding nginx server config blocks
```

Comment:
Note to reviewer: Main change is here, since we are no longer supported custom certs moving forward